### PR TITLE
[NETBEANS-2644] Gradle buildSrc projects are recognized and displayed in the Build Scripts node

### PR DESCRIPTION
**Pull request by lkishalmi**
It looks like something like this:
image

### CODE REVIEW

**Comment by hectorespert** [comment]
It would be good to add the `extide/gradle` module in Travis config to allow Travis run the tests

**Review by ebarboni — APPROVED** [review]
Looks sane to me (not a gradle user) and tested and documented

**Comment by lkishalmi** [comment]
@hectorespert That's valid as actually this one makes some test fails.

File: .travis.yml [hunk 1]
```diff
@@ -335,6 +335,7 @@ matrix:
             - ant $OPTS build
           script:
             - ant $OPTS -f extide/o.apache.tools.ant.module test
+            - ant $OPTS -f extide/gradle test
```
Comment (hectorespert) [inline .travis.yml:338]: 👍

Comment (lkishalmi) [reply .travis.yml]: Well I finally I had to remove this one, as it seems we have a circular cluster dependency on tests, so it fails. I need to fix that in a different PR. But definitely thanks for pointing this out!

**Pull request merged**